Jenkinsfile

```diff
@@ -1,66 +0,0 @@
-properties(
```

Comment:
This file is important.

Reply:
Thanks for noticing, it was a hack space workaround that was meant to be temporary. Reverted